app/src/main/res/layout/fragment_full_number.xml

```diff
@@ -75,7 +75,7 @@
                     android:layout_marginBottom="16dp"
                     android:layout_marginLeft="16dp"
                     android:layout_marginRight="16dp"
-                    android:background="#e5e5e5"
+                    android:background="?attr/colorSurface"
```

Comment:
[minor] could be shortened to ?/colroSurface .. it doesn't need to be explicitly defined